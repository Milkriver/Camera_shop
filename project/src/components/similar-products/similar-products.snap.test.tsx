import {render} from '@testing-library/react';
import { configureMockStore } from '@jedmao/redux-mock-store';
import { Provider } from 'react-redux';
import { createMemoryHistory } from 'history';
import thunk from 'redux-thunk';
import HistoryRouter from '../../components/history-router/history-router';
import SimilarProducts from './similar-products';



describe('Component: SimilarProducts', () => {
  test('should render correctly', () => {
    const middlewares = [thunk];
    const mockStore = configureMockStore(middlewares);

    const store = mockStore({
        OFFERS: {
            offers: [],
        }
    });

    const history = createMemoryHistory();

    const {container} = render(
      <Provider store={store}>
        <HistoryRouter history={history}>
          <SimilarProducts />
        </HistoryRouter>
      </Provider>,
    );

    expect(container).toMatchSnapshot();
  });
});
