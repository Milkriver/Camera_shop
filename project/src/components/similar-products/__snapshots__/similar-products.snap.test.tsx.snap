// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: SimilarProducts should render correctly 1`] = `
<div>
  <section
    class="product-similar"
  />
</div>
`;
